libs/components/forms/src/lib/modules/shared/form-field-hide-when-missing-label.directive.ts

```diff
@@ -0,0 +1,30 @@
+import { Directive, HostBinding, effect, inject, input } from '@angular/core';
+
+import { SkyFormFieldLabelTextRequiredService } from './form-field-label-text-required.service';
+
+/**
+ * Hide form field components of field groups that are missing label text.
+ * @internal
+ */
+@Directive({
+  selector: '[skyFormFieldHideWhenMissingLabel]',
+  standalone: true,
+})
+export class SkyFormFieldHideWhenMissingLabelDirective {
+  readonly #labelTextRequired = inject(SkyFormFieldLabelTextRequiredService, {
```

Comment:
The only thing this service does is log an error when the parent component requires `labelText` to be defined.

Could we move the `validateLabelText` call to this directive's `ngOnInit` so that consuming components wouldn't need any reference to `SkyFormFieldLabelTextRequiredService` at all?

You could rename the directive to `SkyFormFieldLabelTextRequiredDirective` to make the association more clear.